packages/esm-patient-programs-app/src/programs/programs-detailed-summary.component.tsx

```diff
@@ -29,16 +28,12 @@ import {
 import { CardHeader, EmptyState, ErrorState, launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
 import { findLastState, usePrograms } from './programs.resource';
 import styles from './programs-detailed-summary.scss';
+import { ProgramsActionsMenu } from './programs-actions-menu.component';
```

Comment:
I think the correct name for this component and file should be `ProgramsActionMenu` and `programs-action-menu.component` respectively.